packages/espressocash_app/lib/ui/splash_screen.dart

```diff
@@ -13,13 +13,16 @@ class SplashScreen extends StatelessWidget {
   static const route = SplashRoute.new;
 
   @override
-  Widget build(BuildContext context) => CpTheme.black(
-        child: DecoratedBox(
-          decoration: const BoxDecoration(
-            color: CpColors.yellowSplashBackgroundColor,
-          ),
-          child: Center(
-            child: Assets.images.logoIcon.image(height: 101),
+  Widget build(BuildContext context) => Hero(
```

Comment:
@justinenerio can we just disable page transition animation between splash screen and other screens?

Reply:
@ookami-kb updated the code